blueocean-config/src/main/resources/io/jenkins/blueocean/config/BlueOceanConfig/header.jelly

```diff
@@ -0,0 +1,18 @@
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler">
+  <script>
+    (function () {
+      window.$$blueoceanConfig = JSON.parse('${it.blueOceanConfig}');
```

Comment:
@tfennelly I don't have context on jenkinsCIGlobal. So how does other plugin config is added to it? Please post pointer to doc where other plugins add data to it.

>We could add this in the page markup and have a javascript module for reading it. And to the js-extensions stuff in the same way (kill 2 birds).

Sorry, not sure what you mean by this. You mean blueocean-config injects a <div> element with some id and it gets picked up by js-extensions (?)?